pages/common/npm-ls.md

```diff
@@ -0,0 +1,24 @@
+# npm ls
+
+> Print installed packages to `stdout`.
+> More information: <https://docs.npmjs.com/cli/commands/npm-ls>.
+
+- Print all versions of direct dependencies to `stdout`:
+
+`npm ls`
+
+- Print all installed packages including peer dependencies:
+
+`npm ls --all`
+
+- Print dependencies with extended information:
+
+`npm ls --long`
+
+- Print dependencies in parseable format:
+
+`npm ls --parseable`
+
+- Print dependencies in `JSON` format:
```

Comment:
```suggestion
- Print dependencies in JSON format:
```
In this case, it should be written without the backticks, the same rule applies as when writing for example Git instead of `git`. See style guide.